### PR TITLE
new TS1.1 test

### PR DESCRIPTION
**Pull request by gordonguthrie**
Requires https://github.com/basho/riak_kv/pull/1317.

### CODE REVIEW

File: tests/ts_A_select_window_fn_simple.erl [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    DDL = ts_util:get_ddl(),
+    Data = ts_util:get_valid_select_data(),
+    lager:info("DDL being used is ~p~n", [DDL]),
+    lager:info("putting data ~p~n", [Data]),
+    Qry = "select count(myfamily) from GeoCheckin Where time > 1 and time < 10 and myfamily = 'family1' and myseries ='seriesX'",
+
+    Expected = {[<<"COUNT(myfamily)">>], [{8}]},
+    Got = ts_util:ts_query(
+            ts_util:cluster_and_connect(single), TestType, DDL, Data, Qry),
+    gg:format("Got is ~p~n- Expected is ~p~n", [Got, Expected]),
```
Comment (hmmr) [inline tests/ts_A_select_window_fn_simple.erl]: Hello :}

**Comment by javajolt** [comment]
@gordonguthrie Seems like this is just another simple aggregation test, right?  Looks like tests we have except you have the WHERE clause you were Cuthberting about last week.  https://github.com/basho/riak_test/blob/riak_ts/tests/ts_aggregation_simple.erl

**Comment by hmmr** [comment]
@javajolt Apparently, that test assumes a SELECT query can work without WHERE, but it's not the case:

```
================ ts_aggregation_simple failure stack trace =====================
{{assertEqual_failed,[{module,ts_aggregation_simple},
                      {line,54},
                      {expression,"Got"},
                      {expected,10},
                      {value,{1001,
                              <<"no_where_clause: The query must have a where clause.">>}}]},
 [{ts_aggregation_simple,'-verify_aggregation/1-fun-0-',2,
                         [{file,"tests/ts_aggregation_simple.erl"},{line,54}]},
  {ts_aggregation_simple,verify_aggregation,1,
                         [{file,"tests/ts_aggregation_simple.erl"},{line,54}]},
  {ts_aggregation_simple,confirm,0,
                         [{file,"tests/ts_aggregation_simple.erl"},{line,35}]},
  {riak_test_runner,return_to_exit,3,
                    [{file,"src/riak_test_runner.erl"},{line,159}]}]}
================================================================================
```

**Comment by javajolt** [comment]
@hmmr Interesting. Obviously we need to update those tests to have a **WHERE** clause.

**Comment by gordonguthrie** [comment]
This test can probably be dumped as Brett's test covers it

**Comment by hmmr** [comment]
Closing this then, in favour of @javajolt's battery.

**Pull request closed without merging**